swift/Picture.swift

```diff
@@ -100,39 +105,23 @@ class Picture {
         tvg_paint_set_transform(pointer, &matrix)
     }
 
-    /// Crops the picture to a specified rectangle, maintaining its perceived size.
-    ///
-    /// Note: We "crop" the picture by scaling it relative to it's original size.
-    /// This means that we maintain the same perceived size of the picture before and after cropping.
+    /// Stretches the picture to a fit inside a specified rectangle.
     ///
-    /// This behaviour is akin to a "stretch-to-fit" resizing, as the picture's original aspect ratio is not preserved.
-    func crop(_ crop: CGRect) {
-        // Get rid of the unneeded content.
-        let cropShape = tvg_shape_new()
-        tvg_shape_append_rect(
-            cropShape,
-            Float(crop.minX),
-            Float(crop.minY),
-            Float(crop.width),
-            Float(crop.height),
-            0,
-            0
-        )
-        tvg_paint_set_composite_method(cropShape, pointer, TVG_COMPOSITE_METHOD_CLIP_PATH)
-
-        // Calculate the scale ratio of the picture size versus the crop rectangle.
+    /// This behaviour is akin to a "stretch-to-fit" resizing as the picture's original aspect ratio is not preserved.
+    func stretchToFit(_ rect: CGRect) {
```

Comment:
I've renamed this to `stretchToFit` as we're not actually performing any clipping here, we are resizing the image to fit within the crop rectangle boundary.

Reply:
Yep, that's right 😄